scripts/single-node.sh

```diff
@@ -1,45 +1,94 @@
 #!/bin/sh
 
-set -o errexit -o nounset
+# Stop script execution if an error is encountered
+set -o errexit
+# Stop script execution if an undefined variable is used
+set -o nounset
 
-HOME_DIR=$(mktemp -d -t "celestia_app_XXXXXXXXXXXXX")
+CHAIN_ID="private"
+KEY_NAME="validator"
+KEYRING_BACKEND="test"
+COINS="1000000000000000utia"
+DELEGATION_AMOUNT="5000000000utia"
+CELESTIA_APP_HOME="${HOME}/.celestia-appd"
+CELESTIA_APP_VERSION=$(celestia-appd version 2>&1)
 
-echo "Home directory: ${HOME_DIR}"
+echo "celestia-app home: ${CELESTIA_APP_HOME}"
+echo "celestia-app version: ${CELESTIA_APP_VERSION}"
+echo ""
 
-CHAINID="private"
+# Ask the user for confirmation before deleting the existing celestia-app home
+# directory.
+read -p "Are you sure you want to delete: $CELESTIA_APP_HOME? [y/n] " response
 
-# Build genesis file incl account for passed address
-coins="1000000000000000utia"
-celestia-appd init $CHAINID --chain-id $CHAINID --home ${HOME_DIR}
-celestia-appd keys add validator --keyring-backend="test" --home ${HOME_DIR}
-celestia-appd add-genesis-account $(celestia-appd keys show validator -a --keyring-backend="test" --home ${HOME_DIR}) $coins --home ${HOME_DIR}
-celestia-appd gentx validator 5000000000utia \
-	--keyring-backend="test" \
-	--chain-id $CHAINID \
-	--home ${HOME_DIR}
+# Check the user's response
+if [[ $response != "y" ]]; then
+    # Exit if the user did not respond with "y"
+    echo "You must delete $CELESTIA_APP_HOME to continue."
+    exit 1
+fi
 
-celestia-appd collect-gentxs --home ${HOME_DIR}
+echo "Deleting $CELESTIA_APP_HOME..."
+rm -r "$CELESTIA_APP_HOME"
+
+echo "Initializing validator and node config files..."
+celestia-appd init ${CHAIN_ID} \
+  --chain-id ${CHAIN_ID} \
+  --home ${CELESTIA_APP_HOME} \
+  &> /dev/null # Hide output to reduce terminal noise
```

Comment:
[nit]
Does it make sense to only reduce `stdout` to `/dev/null`, and keep `stderr` so that if an error occurs, the user can see the issue?

Something like: 

```suggestion
  1> /dev/null # Hide output to reduce terminal noise
```

Reply:
You're right, it does make a lot of sense to only redirect `stdout`. But for some reason, `celestia-appd init` outputs normal logs to `stderr` so if we only redirect `stdout`, then we see these logs in the terminal...

```
Initializing validator and node config files...
{"app_message":{"auth":{"accounts":[],"params":{"max_memo_characters":"256","sig_verify_cost_ed25519":"590","sig_verify_cost_secp256k1":"1000","tx_sig_limit":"7","tx_size_cost_per_byte":"10"}},"authz":{"authorization":[]},"bank":{"balances":[],"denom_metadata":[{"base":"utia","denom_units":[{"aliases":["microtia"],"denom":"utia","exponent":0},{"aliases":[],"denom":"TIA","exponent":6}],"description":"The native staking token of the Celestia network.","display":"TIA","name":"TIA","symbol":"TIA","uri":"","uri_hash":""}],"params":{"default_send_enabled":true,"send_enabled":[]},"supply":[]},"blob":{"params":{"gas_per_blob_byte":8,"gov_max_square_size":"64"}},"capability":{"index":"1","owners":[]},"crisis":{"constant_fee":{"amount":"1000","denom":"utia"}},"distribution":{"delegator_starting_infos":[],"delegator_withdraw_infos":[],"fee_pool":{"community_pool":[]},"outstanding_rewards":[],"params":{"base_proposer_reward":"0.010000000000000000","bonus_proposer_reward":"0.040000000000000000","community_tax":"0.020000000000000000","withdraw_addr_enabled":true},"previous_proposer":"","validator_accumulated_commissions":[],"validator_current_rewards":[],"validator_historical_rewards":[],"validator_slash_events":[]},"evidence":{"evidence":[]},"feegrant":{"allowances":[]},"genutil":{"gen_txs":[]},"gov":{"deposit_params":{"max_deposit_period":"172800s","min_deposit":[{"amount":"10000000","denom":"utia"}]},"deposits":[],"proposals":[],"starting_proposal_id":"1","tally_params":{"quorum":"0.334000000000000000","threshold":"0.500000000000000000","veto_threshold":"0.334000000000000000"},"votes":[],"voting_params":{"voting_period":"172800s"}},"ibc":{"channel_genesis":{"ack_sequences":[],"acknowledgements":[],"channels":[],"commitments":[],"next_channel_sequence":"0","receipts":[],"recv_sequences":[],"send_sequences":[]},"client_genesis":{"clients":[],"clients_consensus":[],"clients_metadata":[],"create_localhost":false,"next_client_sequence":"0","params":{"allowed_clients":["06-solomachine","07-tendermint"]}},"connection_genesis":{"client_connection_paths":[],"connections":[],"next_connection_sequence":"0","params":{"max_expected_time_per_block":"30000000000"}}},"mint":{"bond_denom":"utia"},"params":null,"qgb":{"params":{"data_commitment_window":"400"}},"slashing":{"missed_blocks":[],"params":{"downtime_jail_duration":"600s","min_signed_per_window":"0.500000000000000000","signed_blocks_window":"100","slash_fraction_double_sign":"0.050000000000000000","slash_fraction_downtime":"0.010000000000000000"},"signing_infos":[]},"staking":{"delegations":[],"exported":false,"last_total_power":"0","last_validator_powers":[],"params":{"bond_denom":"utia","historical_entries":10000,"max_entries":7,"max_validators":100,"min_commission_rate":"0.000000000000000000","unbonding_time":"1814400s"},"redelegations":[],"unbonding_delegations":[],"validators":[]},"transfer":{"denom_traces":[],"params":{"receive_enabled":true,"send_enabled":true},"port_id":"transfer"},"vesting":{}},"chain_id":"private","gentxs_dir":"","moniker":"private","node_id":"07004beac9d706bf742146cd21469f3081ce17b3"}
```

Reply:
is this the same for all commands? maybe we should check how we're initializing the outputs. I can investigate later of needed

Reply:
Yes it's the same for all commands. I don't see a setting in this script that is redirecting stderr to stdout globally so maybe the celestia-appd binary itself emits all output to stderr instead of stdout?